package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@nearform/heap-profiler",
+  "version": "0.1.0",
```

Comment:
```suggestion
  "version": "0.0.1",
```

Bump a minor directly on master and release.

Reply:
Will do.